api/bin/www.js

```diff
@@ -4,57 +4,62 @@
  * Module dependencies.
  */
 
-import app from "../app.js"
-import debug from "debug"
-import http from "http"
-import mongoose from "mongoose"
-import { MONGO_URI } from "../store.js"
+import app from '../app.js';
+import debug from 'debug';
+import http from 'http';
+import mongoose from 'mongoose';
+import { MONGO_URI } from '../store.js';
 
-const serverDebug = debug("server:server")
+const serverDebug = debug('server:server');
 /**
  * Get port from environment and store in Express.
  */
 
-const port = normalizePort(process.env.PORT || "8000")
-app.set("port", port)
+const port = normalizePort(process.env.PORT || '8000');
+app.set('port', port);
 
 /**
  * Create HTTP server.
  */
 
-const server = http.createServer(app)
+const server = http.createServer(app);
 
 /**
  * Listen on provided port, on all network interfaces.
  */
-mongoose.set("strictQuery", true)
+mongoose.set('strictQuery', true);
 // mongodb://127.0.0.1:27017/upkeepdev
 
 process.env.MONGOURI
-	? mongoose
-			.connect(process.env.MONGOURI)
-			.then(() => {
-				server.listen(port)
-				server.on("error", onError)
-				server.on("listening", onListening)
-				console.log(`Server listening and ${process.env.MONGOURI} connected !!`)
-			})
-			.catch((err) => {
-				console.log(err)
-				process.exit(0)
-			})
-	: mongoose
-			.connect(MONGO_URI)
-			.then(() => {
-				server.listen(port)
-				server.on("error", onError)
-				server.on("listening", onListening)
-				console.log("🚀 Listening and MongoDB connected ✅ in port", port)
-			})
-			.catch((err) => {
-				console.log(err)
-				process.exit(0)
-			})
+    ? mongoose
```

Comment:
Ok nice way of handling different mongo database

Reply:
Thanks